modules/edit-modes/test/lib/__snapshots__/measure-angle-mode.test.ts.snap

```diff
@@ -1,9 +1,9 @@
-// Jest Snapshot v1, https://goo.gl/fbAQLP
+// Vitest Snapshot v1
 
-exports[`three clicks + pointer move tooltip contains angle 1`] = `
-Array [
```

Comment:
Why is this changing? Some jest version upgrade? Can this be handled in a separate PR?

Reply:
> Some jest version upgrade? 
[vitest](https://vitest.dev/) it's compatible replacement for jest. 
[Here](https://vitest.dev/guide/comparisons.html#jest) is a description of why jest does not work as well in vite as vitest